Library/Formula/cowpatty.rb

```diff
@@ -12,6 +12,8 @@ class Cowpatty < Formula
   end
 
   def install
+    # May fail on parallel builds with a race condition, see #45150
```

Comment:
It'd be good to get something upstream, but feel free to merge with the current comment if there's no public way of accessing the email. Probably worth pulling bottles as well, given the `make` stage.